android/src/main/java/com/swmansion/rnscreens/ScreenStack.java

```diff
@@ -341,22 +362,29 @@ private void setupBackHandlerIfNeeded(ScreenStackFragment topScreen) {
         break;
       }
     }
+
     if (topScreen != firstScreen && topScreen.isDismissable()) {
       mFragmentManager
-              .beginTransaction()
-              .show(topScreen)
-              .addToBackStack(BACK_STACK_TAG)
-              .setPrimaryNavigationFragment(topScreen)
-              .commitAllowingStateLoss();
+          .beginTransaction()
+          .show(topScreen)
+          .addToBackStack(BACK_STACK_TAG)
+          .setPrimaryNavigationFragment(topScreen)
+          .commitAllowingStateLoss();
       mFragmentManager.addOnBackStackChangedListener(mBackStackListener);
     }
   }
 
   private static boolean isCustomAnimation(Screen.StackAnimation stackAnimation) {
```

Comment:
After you've inverted the logic here it should no longer be named `isCustomAnimation`.